com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -2,99 +2,101 @@
 
 [High Dynamic Range](https://en.wikipedia.org/wiki/High-dynamic-range_imaging) content has a wider color gamut and greater luminosity range than standard definition content.
 
-HDRP can output HDRP content for devices which support that functionality.
+HDRP can output HDR content for displays which support that functionality.
 
 ## Enabling HDR Output
 
-To enable HDR output, navigate to **Project Settings > **Player** > **Other Settings** and enable **Use display in HDR mode**.
+To activate HDR output, navigate to **Project Settings > Player** > **Other Settings** and enable **Use display in HDR mode**.
 
-## Configuring Tonemapping settings for HDR displays
+## HDR tone mapping in HDRP
 
-Enable **Use display in HDR mode** to reveal HDR-related options in the [Tonemapping](https://github.com/Unity-Technologies/Graphics/pull/Post-Processing-Tonemapping.md) [Volume](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@13.0/manual/Volumes.html) component.
+After you enable **Use display in HDR mode**, you must configure [Tonemapping](https://github.com/Unity-Technologies/Graphics/pull/Post-Processing-Tonemapping.md) settings for your HDR input.
```

Comment:
Why is this link to a pull?

Reply:
@FrancescoC-unity  just checking in about this change.

Reply:
Yeah apologies I am super swamped atm, will try to make it a priority tomorrow morning.

Reply:
no worries, just good to know you saw these.